packages/network/nfs-utils/package.mk

```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2024-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="nfs-utils"
+PKG_VERSION="2.6.4"
+PKG_SHA256="01b3b0fb9c7d0bbabf5114c736542030748c788ec2fd9734744201e9b0a1119d"
+PKG_LICENSE="GPL-2.0-or-later"
```

Comment:
Are you sure its "or later" and not just "2.0-only" ?? Saw other pointers for 2.0-only but not really saw a valid source for it.
Sadly they just copy pasta the licence file (same for "and later" and only 2.0) and not really declare it :/

Reply:
The mount code and most of the other code are 2-or-later https://git.linux-nfs.org/?p=steved/nfs-utils.git;a=blob;f=utils/mount/mount.c;h=b98f9e00eb887ef82cc72459ccb4093639e64f29;hb=HEAD

Only the junction code (which we don't even enable) and a few other bits are 2-only so I think 2-or-later is fine